website/templates/emails/collection_submission_submitted.html.mako

```diff
@@ -8,7 +8,7 @@
         Hello ${user.fullname},<br>
         <br>
         <p>
-            % if is_initator:
+            % if is_initiator:
```

Comment:
Note to myself (and any other reviewer), this is updated in `[osf/models/collection_submission.py](https://github.com/CenterForOpenScience/osf.io/pull/11379/files#diff-204ae98386ab64d3648e802df27581f8d252b275a7d596ba2009c2b88191ade9) as well